db/seeds.rb

```diff
@@ -9,128 +9,128 @@
                              categories: [seafood_cat, entree_cat])
 
 minotaur       = Item.create(title: "NY Strip Minotaur",
-                             description: "Gooooood",
+                             description: "A dish served as rare as the beast itself",
                              price: 19.99,
                              status: "active",
                              categories: [meat_cat, entree_cat])
 
 scylla         = Item.create(title: "Scyllamari",
-                             description: "Gooooood",
+                             description: "Trust us when we say this dish isn't 'Charybian'",
                              price: 17.50,
                              status: "retired",
                              categories: [seafood_cat, entree_cat])
 
 griffin_wings  = Item.create(title: "Buffalo Griffin Wings",
-                             description: "Gooooood",
+                             description: "These little wings pack an especially powerful--maybe even majestic--punch",
                              price: 6.00,
                              status: "active",
                              categories: [meat_cat, appetizer_cat])
 
 pegasus_wings  = Item.create(title: "Buffalo Pegasus Wings",
-                             description: "Gooooood",
+                             description: "One little bit will set your tastebuds aflight",
                              price: 6.00,
                              status: "retired",
                              categories: [meat_cat, appetizer_cat])
 
 leg_of_griffin = Item.create(title: "Leg of Griffin",
-                             description: "Gooooood",
+                             description: "Suffice it to say the king of beasts makes a meal fit for kings",
                              price: 9.50,
                              status: "active",
                              categories: [meat_cat, entree_cat])
 
 leg_of_pegasus = Item.create(title: "Leg of Pegasus",
-                             description: "Gooooood",
+                             description: "This dish will send you galloping off to tasty bliss",
                              price: 9.50,
                              status: "retired",
                              categories: [meat_cat, entree_cat])
 
 hydra          = Item.create(title: "Hydra Vodka",
-                             description: "Gooooood",
+                             description: "Careful with this one! We serve it so virulent your friends may appear to have many heads",
                              price: 6.00,
                              status: "active",
                              categories: [drinks_cat])
 
-rainbow_pie          = Item.create(title: "Rainbow Pie",
-                             description: "Gooooood",
+rainbow_pie          = Item.create(title: "Ogre Pie",
+                             description: "We hope your appetite is as voracious as this dish's namesake",
```

Comment:
Line is too long. [106/80]<br>Align the elements of a hash literal if they span more than one line.